source/_integrations/rtsp_to_webrtc.markdown

```diff
@@ -0,0 +1,42 @@
+---
+title: RTSPtoWebRTC
+description: Instructions on how to integrate RTSPtoWebRTC server within Home Assistant.
+ha_category:
+  - Other
+ha_config_flow: true
+ha_release: 2022.2
+ha_iot_class: Local Push
+ha_quality_scale: internal
+ha_codeowners:
+  - '@allenporter'
+ha_domain: rtsp_to_webrtc
+---
+
+The `rtsp_to_webrtc` integration registers with `camera` to provide WebRTC live streams for any RTSP camera. The integration initiates a connection to a [RTSPtoWeb](https://github.com/deepch/RTSPtoWeb) or [RTSPtoWebRTC](https://github.com/deepch/RTSPtoWebRTC) proxy server that converts an RTSP stream to a WebRTC stream.
```

Comment:
We used to backtick integration names in the days it was used in YAML configuration... that is no longer the case; so we can just write the names in full.
```suggestion
The RTSPtoWebRTC integration registers with [camera integration](/integration/camera) to provide WebRTC live streams for any RTSP camera. The integration initiates a connection to a [RTSPtoWeb](https://github.com/deepch/RTSPtoWeb) or [RTSPtoWebRTC](https://github.com/deepch/RTSPtoWebRTC) proxy server that converts an RTSP stream to a WebRTC stream.
```